examples/src/utils/CadBudgetUi.ts

```diff
@@ -11,7 +11,7 @@ import dat from 'dat.gui';
  */
 export function initialCadBudgetUi(viewer: Cognite3DViewer, uiFolder: dat.GUI) {
   const state = {
-    factor: 100.0
+    factor: 400.0
```

Comment:
I suppose this might be why you have a lower frame rate than I had - perhaps it can be reverted to a middle ground? 200?

Reply:
Fixed